components/Home/TableRow.js

```diff
@@ -0,0 +1,17 @@
+export default function TableRow({ Position, Salary, Date, Link }) {
+  return (
+    <tr>
+      <td className="p-4 font-semibold">{Position}</td>
+      <td>{Salary}</td>
+      <td>{Date}</td>
+      <td className="p-4">
```

Comment:
remove left padding from the apply button so it will be aligned with the heading